suite-native/navigation/src/navigators.ts

```diff
@@ -116,9 +120,12 @@ export type RootStackParamList = {
     };
     [RootStackRoutes.DevUtilsStack]: undefined;
     [RootStackRoutes.AccountDetail]: {
-        accountKey: AccountKey;
+        accountKey?: AccountKey;
         tokenContract?: TokenAddress;
-    };
+    } & RequireAllOrNone<
+        { networkSymbol?: NetworkSymbol; accountType?: AccountType; accountIndex?: number },
+        'networkSymbol' | 'accountType' | 'accountIndex'
+    >;
```

Comment:
When you use `RequireAllOrNone`, doesn't it make more sense to be like this? 
```suggestion
    } & RequireAllOrNone<
        { networkSymbol: NetworkSymbol; accountType: AccountType; accountIndex: number },
        'networkSymbol' | 'accountType' | 'accountIndex'
    >;
```

Reply:
ab1ff66d66